src/ldp/alg/callbacks.py

```diff
@@ -94,6 +104,13 @@ async def after_agent_get_asv(
     ) -> None:
         """Invoked by runners after agent.get_asv()."""
 
+    @asynccontextmanager
+    async def during_env_step(
```

Comment:
Nice work.

Do you mind extending `test_rollouts.DummyCallback` for these in the unit tests?